java/vector/src/main/java/org/apache/arrow/vector/complex/BaseRepeatedValueVector.java

```diff
@@ -355,16 +355,12 @@ public int getInnerValueCountAt(int index) {
             offsetBuffer.getInt(index * OFFSET_WIDTH);
   }
 
-  /** Return if value at index is null (this implementation is always false). */
+  /** Return if value at index is null. */
   @Override
-  public boolean isNull(int index) {
-    return false;
-  }
+  public abstract boolean isNull(int index);
```

Comment:
isNull is just part of the ValueVector interface, so we should be able to get rid of this declaration entirely.

Reply:
Yes that works better I guess.